tests/test_processor.py

```diff
@@ -126,7 +126,7 @@ def test_granularity(self):
 class TestRepository(TestCase):
     def setUp(self):
         self.subject = RepositoryFactory.build()
-        self.date_str = "2015-07-05T22:16:18+00:00"
+        self.date_str = "2015-07-05 22:16:18+00:00"
```

Comment:
If we use `datetime_obj.isoformat()` instead of `str(datetime_obj)` we can keep the `T`, as that is how the services send us the data.